documentation/asciidoc/topics/yaml/identities_batch.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: connect-secret
+type: Opaque
+stringData:
+  # The key must be 'identities-batch'.
+  # The content is "user create" commands for the {brandname} CLI.
+  identities-batch: |-
```

Comment:
I think it's worth adding the metrics keys and explaining their purpose in this example so something like:

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: connect-secret
type: Opaque
stringData:
  # The user account that will be used for authentication by the Prometheus ServiceMonitor
  username: monitor
  # The password associated with the 'username' account define above
  password: password
  # The key must be 'identities-batch'.
  # The content is "user create" commands for the {brandname} CLI.
  identities-batch: |-
    user create user1 -p changeme -g admin
    user create user2 -p changeme -g deployer
    user create monitor -p password --users-file metrics-users.properties --groups-file metrics-groups.properties
```